dvc/stage.py

```diff
@@ -197,7 +214,12 @@ def is_stage_file(path):
         return os.path.isfile(path) and Stage.is_valid_filename(path)
 
     def changed_md5(self):
-        return self.md5 != self._compute_md5()
+        for h in self.hash:
```

Comment:
There is no need to change hash when computing checksum for the stage file itself. md5 is fine for that application.

Reply:
Though, thinking about it, it does kinda make sense. Ok, let's keep this logic for now.

Reply:
Thanks